hooks/hookexecution/executor_test.go

```diff
@@ -699,7 +699,6 @@ func TestExecuteRawAuctionStage(t *testing.T) {
 
 func TestExecuteProcessedAuctionStage(t *testing.T) {
 	foobarModuleCtx := &moduleContexts{ctxs: map[string]hookstage.ModuleContext{"foobar": nil}}
-	account := &config.Account{}
```

Comment:
Do we want to remove this from every test in this file?

Reply:
I went through and tried removing all of these lines, and the tests still passed, so I think it can be removed.

Reply:
I agree with Alex; we can remove them.